### PR TITLE
Made channel select the same in all viewports.

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

Made channel select the same in all viewports. Also some cleanup.

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/vue/toolbar/index.vue [hunk 1]
```diff
@@ -1,22 +1,14 @@
 <template>
 
-  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" v-show='!searchOpen' >
+  <div v-bind:class="['toolbar', displayToolbar ? 'toolbar-hide' : '']" v-show='!searchOpen'>
     <breadcrumbs class="breadcrumbs"></breadcrumbs>
-    <div :class="{ 'toggle-menu-on' : more }">
-      <label for="chan-select" :class="[ more ? 'label-on' : 'visuallyhidden' ]" >{{ $tr('switchChannels') }}</label>
```
Comment (radinamatic) [inline kolibri/plugins/learn/assets/src/vue/toolbar/index.vue:6]: @christianmemije I may be missing something, but where did the `<label>` for this `select` end up?

Comment (christianmemije) [inline kolibri/plugins/learn/assets/src/vue/toolbar/index.vue:6]: @radinamatic I removed it. Quick google search shows that this is necessary for accessibility. My mistake. Should the label should be visible or can be just part of the markup but hidden?

Comment (radinamatic) [inline kolibri/plugins/learn/assets/src/vue/toolbar/index.vue:6]: I must have been doing a banged up job so far, if you had to google it... :worried: 

It had the global `visuallyhidden` class so far, so you can apply it again!

File: kolibri/plugins/learn/assets/src/vue/toolbar/index.vue [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
+      <label for="chan-select" class="visuallyhidden">{{ $tr('switchChannels') }}</label>
       <select
-        class="chan-select"
-        :class="[ more ? 'chan-select-mobile-location' : 'chan-select-location' ]"
-        id="chan-select"
         name="chan-select"
+        class="chan-select"
```
Comment (radinamatic) [inline kolibri/plugins/learn/assets/src/vue/toolbar/index.vue:9]: Select will need an `id` to match the `for` in `<label for="chan-select">`.

@christianmemije

**Comment by radinamatic** [comment]
Good to go! :+1:

File: kolibri/plugins/learn/assets/src/vue/toolbar/index.vue [hunk 3]
```diff
@@ -78,15 +66,12 @@
         this.more = false;
 
         if (this.currScrollTop > this.lastScrollTop) {
-          this.displayToolBar = true;
+          this.displayToolbar = true;
         } else {
-          this.displayToolBar = false;
+          this.displayToolbar = false;
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/vue/toolbar/index.vue:71]: nice

**Comment by indirectlylit** [comment]
much cleanup

**Pull request merged**